src/uct/tcp/tcp.h

```diff
@@ -193,10 +193,10 @@ enum {
  * TCP connection request packet
  */
 typedef struct uct_tcp_cm_conn_req_pkt {
-    uct_tcp_cm_conn_event_t       event;      /* Connection event ID */
-    uint8_t                       flags;      /* Packet flags */
-    struct sockaddr_in            iface_addr; /* Socket address of UCT local iface */
-    uct_tcp_ep_cm_id_t            cm_id;      /* EP connection mananger ID */
+    uct_tcp_cm_conn_event_t event;         /* Connection event ID */
+    uint8_t                 flags;         /* Packet flags */
+    uct_tcp_ep_cm_id_t      cm_id;         /* EP connection mananger ID */
+    /* Socket address follows. The address size is according to the address family in use. */
```

Comment:
could you please ensure that this comment fits "80 symbols per line" rules, e.g. by breaking the line?